variants/frontend-base/template.rb

```diff
@@ -1,6 +1,9 @@
 source_paths.unshift(File.dirname(__FILE__))
 
 run "mv app/assets app/frontend"
+run "mkdir app/assets"
+run "mv app/frontend/config app/assets/config"
```

Comment:
This  moves `app/assets/config/manifest.js` back to the place Sprockets requires it.

Reply:
Is that just Sprockets? I did find putting things in `app/frontend` to be by far the most brittle part of this template, so I wonder if we should just stick with Rails defaults.

Reply:
Sprockets hard codes this path. https://github.com/rails/sprockets-rails/issues/369 is open to see if it can be made customisable. I take your point on the defaults. I'm loathe to give it up because the default feels gross but accepting the default might be the most pragmatic.